.github/pull_request_template.md

```diff
@@ -0,0 +1,31 @@
+Tag name (required for release branches):
```

Comment:
Maybe modify to say `(required for commits which will be moved immediately to release branch)`

Reply:
I *think* the `(required for release branches)` is for PRs to main (or a release branch), rather then for PRs that are targeted for development but are destined for release. Does that make sense?

Reply:
I think that the commits happen on the development branch and then the main branch doesn't actually have commits, but rather merges.  Perhaps I'm thinking of a different repo's workflow where this happens.  @nusbaume - where did Haipeng have his commit not having the tagname because it was committed in the branch and he didn't have an opportunity to insert it later.

Reply:
It was in atmospheric physics I think so the tag name had to be filled in by a future commit.

Reply:
Yeah I ended up filling in @jimmielin's tag name later.  I agree with @cacraigucar's wording, maybe we can change it slightly to something like `(required for the last development PR before merging into a release branch)`?

Reply:
Actually scratch that last comment.  I think we can leave it as just `(required for release branches)` because for CAM-SIMA the release branch will always have a PR template entry, even for merge commits, so we can just add the tag name in then.